wiki/People/Global_Moderation_Team/zh.md

```diff
@@ -74,16 +72,18 @@ tags:
 
 [全局管理团队 (GMT) 成员列表](https://osu.ppy.sh/groups/4)列出了所有在任团队成员。除了下面提到的领域外，所有全局管理团队 (GMT) 成员都能够[审查谱面内的音频或视觉内容](/wiki/Rules/Visual_content_considerations)。
 
+**无论 GMT 成员处于哪个类别，他们都能参与其他类别下的任何任务。** 例如，主要负责管理聊天的 GMT 成员也可以在与管理论坛相关的事务中提供帮助。反之亦然。
+
 <!-- NOTE FOR TRANSLATORS: Translating this section is different from other parts of the osu! wiki. See https://github.com/ppy/osu-wiki/blob/master/meta/group-info/TRANSLATING.md#group-articles. -->
 
 | 名字 | 语言 | 专职范围 |
 | :-- | :-- | :-- |
 | ::{ flag=MY }:: [\_Kobii](https://osu.ppy.sh/users/6209713) | 汉语，马来语，粤语，一部分日语 | 作图摸图社群管理员 |
 | ::{ flag=DE }:: [- Felix](https://osu.ppy.sh/users/8503985) | 德语 | 聊天管理员 |
 | ::{ flag=CH }:: [\[ryuu\]](https://osu.ppy.sh/users/5698467) | 俄语 | 聊天管理员 |
-| ::{ flag=RS }:: [0x84f](https://osu.ppy.sh/users/7944724) | 塞尔维亚语 | General moderation |
+| ::{ flag=RS }:: [0x84f](https://osu.ppy.sh/users/7944724) | 塞尔维亚语 | 全局管理员 |
```

Comment:
this (lack of) translation comes from `meta/group-info/zh.yaml` fwiw. it will be overridden at some point if you don't also update that file

Reply:
Is it okay to update the yaml file in this pr? or do I need to open a new one?

Reply:
normally separate is better, but you already have all the group updates in this one so I guess it would be fine here too